.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,41 @@
+# Standard GPflow pull request
+
+At first, thank you very much for spending time on contributing to GPflow.\
+This template exists to help us to grasp your work without asking very common and questions. The template helps you to get thorough feedback as well.
```

Comment:
"common and questions" <- did you miss a word before and? by itself the "and" doesn't make sense